### DISCUSSION TITLE
2005/05/03/%E8%AF%95%E4%B8%80%E4%B8%8B%E4%BD%BF%E7%94%A8%E8%83%8C%E6%99%AF%E5%9B%BE%E6%A1%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#17] [Announcements]
# 2005/05/03/%E8%AF%95%E4%B8%80%E4%B8%8B%E4%BD%BF%E7%94%A8%E8%83%8C%E6%99%AF%E5%9B%BE%E6%A1%88

The prelude of the night is commenced in the music of the sunset,

https://blog.qizhen.xyz/2005/05/03/%E8%AF%95%E4%B8%80%E4%B8%8B%E4%BD%BF%E7%94%A8%E8%83%8C%E6%99%AF%E5%9B%BE%E6%A1%88

**Comment by ruanqizhen** [comment]
Qizhen
2005年05月5日 12:14 编辑
另外，欢迎大家转载！

回复
Qizhen
2005年05月5日 11:57 编辑
这是我同事拍得，我当时没带相机，否则一定比这漂亮 :)Lake Travis, Auistin, TX

回复
Unknown
2005年05月5日 11:43 编辑
夕阳无限好哇，照片很美，是阁下的大作吗？

回复
小＆U
2005年05月4日 20:26 编辑
这张照片蛮灵的~

回复
